### DISCUSSION TITLE
Discord be gone

### DISCUSSION THREAD

**Discussion by BoredFishRE** [ProdigyPNP/ProdigyMathGameHacking#192] [Forum]
Discord's dead as hell

**Comment by afkvido** [comment]
@BoredFishRE My Discord account got terminated

**Reply by BoredFishRE** [reply]
did they say why lol

**Reply by eris-webserv** [reply]
> did they say why lol

Prodigy probably terminated them.
Most signs point to it:
- Hacks broke A G A I N, telling me Prodigy put in a patch
- Terminations usually happen due to mass reports, or **DMCAs**

**Reply by afkvido** [reply]
(screenshot is massive)

**Reply by BoredFishRE** [reply]
> (screenshot is massive)

ah yeah probably the zoom bombing. tbh you had it coming.

**Reply by vibinyama** [reply]
> ![image](https://user-images.githubusercontent.com/69060894/197423035-be364c24-b38b-421b-9bbb-363eff68727f.png)

Tbh major L maybe don’t be hella weird next time

**Reply by afkvido** [reply]
imma see if i can switch to something that isn't dsicord primarily, but keep a second discord for reasons such as.... a bridge

**Reply by afkvido** [reply]
also time to report the Prodigy X server, since that alliance is gone

**Reply by ashtoncoded** [reply]
Same

**Reply by afkvido** [reply]
wdym "Same" @AgrTheDev lol

**Reply by ashtoncoded** [reply]
Reporting them rn

**Reply by afkvido** [reply]
Why? Aren't you running Prodigy X?
